catalog/nsili/catalog-nsili-transformer/src/main/java/org/codice/alliance/nsili/transformer/DAGConverter.java

```diff
@@ -91,6 +93,32 @@
 
   private MetacardType nsiliMetacardType;
 
+  private static Map<String, String> typeConversionMap;
+
+  static {
+    typeConversionMap = new HashMap<>();
+    typeConversionMap.put("COLLECTION/EXPLOITATION PLAN", "Collection");
+    typeConversionMap.put("DOCUMENT", "Text");
+    typeConversionMap.put("GEOGRAPHIC AREA OF INTEREEST", "Dataset");
+    // typeConversionMap.put("GEOSPATIAL VECTOR", "");
+    // typeConversionMap.put("GMTI", "");
```

Comment:
why are these commented out?

Reply:
Sounds like a combination of
1. The mapping NSIL -> DCMI wasn't clear
2. The mapping wasn't needed for the data being handled when this was written so it was ignored. 

Created https://codice.atlassian.net/browse/CAL-520 to address this, and added a TODO here